sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/NullDefaultUtility.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure.spi;
+
+import javax.annotation.Nullable;
+
+/**
+ * Holder for the non-public defaultIfNull method. This serves only to mitigate the method being on
+ * a public interface.
+ */
+final class NullDefaultUtility {
```

Comment:
Can call this `ConfigUtil` to reduce friction when adding more methods for the same reason